packages/docs/docs/docs/first.md

```diff
@@ -0,0 +1,161 @@
+
+```solidity
+pragma solidity ^0.5.0;
+
+contract Counter {
+  uint256 public value;
+  
+  function increase() public {
+    value++;
+  }
+}
+```
+
+This contract will just keep a numeric `value`, that will be increased by one every time we send a transaction to the `increase()` function.
+
+You can run `npx zos compile` to compile the contract and check for any errors. After it compiled successfully, we can now deploy our contract.
+
+> Note: You don't have to worry if you forget to compile your contract. The CLI will automatically check if your contract changed when you run any command, and compile it if needed.
+
+## Deploying to a development network
+
+We will use [ganache](https://truffleframework.com/ganache) as a _development network_ to deploy our contract. If you don't have ganache installed, do so now by running `npm install -g ganache-cli`.
+
+Development networks are mini blockchains that run just on your computer, and are much faster than the actual Ethereum network. We will use one for coding and testing.
+
+Open a separate terminal, and start a new ganache process:
+
+```console
+ganache-cli --deterministic
+```
+
+This will start a new development network, using a deterministic set of accounts, instead of random ones. We can now deploy our contract there, running `npx zos create`, and choosing to deploy the `Counter` contract to the `local` network.
+
+```console
+$ npx zos create
+✓ Compiled contracts with solc 0.5.9 (commit.e560f70d)
+? Pick a contract to instantiate: Counter
+? Pick a network: local
+✓ Added contract Counter
+✓ Contract Counter deployed
+? Do you want to call a function on the instance after creating it?: No
+✓ Setting everything up to create contract instances
+✓ Instance created at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+```
+
+> Note: The addresses where your contracts are created, or the transaction identifiers you see, may differ from the ones listed here.
+
+Our counter contract is deployed to the local network and ready to go! We can test it out by interacting with it from the terminal. Let's try incrementing the counter, by sending a transaction to call the `increase` function through `npx zos send-tx`.
+
+```console
+$ npx zos send-tx
+? Pick a network: local
+? Pick an instance: Counter at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+? Select which function: increase()
+✓ Transaction successful. Transaction hash: 0x20bef6583ea32cc57fe179e34dd57a5494db3c403e441624e56a886898cb52bd
+```
+
+We can now use `npx zos call` to query the contract's public `value`, and check that it was indeed increased from zero to one.
+
+```console
+$ npx zos call
+? Pick a network: local
+? Pick an instance: Counter at 0xCfEB869F69431e42cdB54A4F4f105C19C080A601
+? Select which function: value()
+✓ Method 'value()' returned: 1
+```
+
+<!-- We could move the following to a separate tutorial -->
+## Updating your contract
```

Comment:
Same. As we are referencing the `upgrade` command, I would change it to `Upgrade`